Gemfile

```diff
@@ -1,6 +1,5 @@
 source 'https://rubygems.org'
 
-
```

Comment:
if you're going to be using this app as the workspace, please make sure gem version matches with whats in LTP 5.